### DISCUSSION TITLE
How to insert a TABLE ValueType into a SR?

### DISCUSSION THREAD

**Discussion by Simontjunjie** [nroduit/Weasis#606] [Q&A]
The DICOM standard introduced a new value type called TABLE for SR in 2020. I have filled in the required tags according to the specifications, but when I open the file with Weasis, the table does not display. I would like to ask if anyone has written SR containing TABLE type?, or does anyone have a DICOM file of SR that includes the TABLE type?
Thanks

**Comment by Simontjunjie** [comment]
Weasis DICOM SR Viewer Display "TABLE is not supported!"What it means?  Is it that my SR is incorrect, or does Weasis currently not support TABLE display?

**Comment by nroduit** [comment]
The SR view was implemented a few years ago, and some types had not been implemented, such as TCOORD and WAVEFORM. The same applies to new elements such as SCOORD3D and TABLE.
